service/worker/scheduler/workflow_test.go

```diff
@@ -1245,6 +1245,59 @@ func (s *workflowSuite) TestBackfill() {
 	)
 }
 
+func (s *workflowSuite) TestBackfillInclusiveStartEnd() {
```

Comment:
you can check `currentTweakablePolicies.Version` and `Skip()` here so the test works before and after the change